src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -251,6 +222,8 @@ def step(self, t, t_new, current_rv):
     def rvlist_to_odesol(self, times, rvs):
         """Create an ODESolution object."""
 
+        rvs = self.diffusion.postprocess_states(rvs, times)
```

Comment:
this happens in this function, not in postprocess, because it needs access to only the RV list. I found this more convenient this way.

Reply:
It is a bit confusing to have such functionality in `rvlist_to_odesol`, it feels like it should rather go into "postprocess".

Reply:
Hmmm okay. Do you think rvlist_to_odesol should happen before or after postprocess?

Reply:
Anyway, changed it.